frontend/Controls/ModuleMenuButton.qml

```diff
@@ -36,11 +42,45 @@ Rectangle {
         cursorShape: Qt.PointingHandCursor
 
         onClicked: {
-            selectView(target);
+            selectView(target)
         }
 
         hoverEnabled: true
-        onEntered: { button.state = 'hover' }
-        onExited: { button.state = '' }
+        onHoveredChanged: {
+            button.state = containsMouse ? "hover" : ""
+        }
+        /*
```

Comment:
Mind removing the commented out code? keeps things tidy :)

Reply:
Ok ;)